observer.go

```diff
@@ -101,8 +101,11 @@ func newObserver(server *aero.Host, user, pass string) (o *Observer, err error)
 	clientPolicy := aero.NewClientPolicy()
 	clientPolicy.User = string(username)
 	clientPolicy.Password = string(password)
+
 	if authMode == "external" {
 		clientPolicy.AuthMode = aero.AuthModeExternal
+	} else if authMode == "pki" {
+		clientPolicy.AuthMode = aero.AuthModePKI
```

Comment:
```suggestion
        switch authMode {
        case "internal":
		clientPolicy.AuthMode = aero.AuthModeInternal
        case "external":
		clientPolicy.AuthMode = aero.AuthModeExternal
	case "pki":
		clientPolicy.AuthMode = aero.AuthModePKI
        default:
		log.Fatalln("Invalid auth mode: only `internal`, `external`, `pki` values are accepted.")
    }
```
and remove line 85-86

Reply:
Fixed.  I also added the default value of "" as an option.